### ISSUE TITLE
Screen should follow tab highlight

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balloons-and-static-electricity#551]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/721
When tabbing around objects while zoomed in, if the object being focused on is off screen, the screen does not pan to see that object.

**Visuals**
![taboffscreen](https://user-images.githubusercontent.com/41024075/138493048-7fe3f4fe-8712-4edb-9a41-e56075d19d3c.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬:‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.2/phet/balloons-and-static-electricity_all_phet.html?stringTest=double
Version: 1.5.0-rc.2 2021-10-11 21:55:48 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
Looks good in rc.3
